src/wp-includes/functions.php

```diff
@@ -6998,6 +6998,27 @@ function wp_debug_backtrace_summary( $ignore_class = null, $skip_frames = 0, $pr
  * @return int[] Array of IDs not present in the cache.
  */
 function _get_non_cached_ids( $object_ids, $cache_key ) {
+	$object_ids_filter = static function ( $object_id ) {
```

Comment:
I am not sure we need to make this   Asynchronous function. Why not make a helper function and add unit tests to that.

Reply:
What's the function called and would it call `_doing_it_wrong()`?

Reply:
Something like `_sanaitize_cache_ids`? and it would be the same logic as current.

Reply:
Added 👍